helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -401,6 +401,22 @@ logsCollection:
     # All other logs will be ignored.
     useSplunkIncludeAnnotation: false
 
+# Configuration for collecting journald logs using otel collector
+  journald:
+    enabled: true
+    directory: /run/log/journal
```

Comment:
"The journal service stores log data either persistently below `/var/log/journal` or in a volatile way below `/run/log/journal/`"

for better usability, can we have both fields as default? 
I know that journald receiver doesn't support array of `directory` parameter. 
so it may mean building 2 journald receiver, one for each directory unless there is a nice way to find which directory exists.

Reply:
I agree that `/var/log/journal` is more common. I never saw anything under `/run/log/journal/`.

How collector with journald receiver behaves if provided directory is not present?

Ideally this should be solved in journald receiver code, it can have some kind of discovery if the field is not specified.

I don't think two receiver is a good workaround. As far as this field easily configurable, we just need to make sure user can read a warning message and update field if needed. By default I would keep `/var/log/journal` as most common to my knowledge.

WDYT?

Reply:
`How collector with journald receiver behaves if provided directory is not present?` - Currently I think this is a required field for journald to work properly, if wrong directory path is provided, it will not work, it doesn't even error out I think. 

For now, we can do the following: 
- Update default to `/var/log/journald` and add a note to update it to /run/log/journal OR any other value if different? 
- Make it a required field, and comment it out so user has to provide an input for this field for journald to work?
- If we can check which one is available and acc update template with that directory path (not sure if there is an easy way to discover which path is present) 
- add 2 separate receivers?
- or if any other approach?

Reply:
The only requirement of this helm chart is that it must work OOTB with only two fields set in `splunkObservability` or/and `splunkPlatform`. We cannot introduce any other required fields like directory for journald. To make things simple and make this merged sooner I see only two reasonable options:
1. Disable journald by default and make the directory setting required. So if customer enables the journald they have to provide the path.
2. Keep journald enabled with most commonly used directory, which is `/var/log/journald` I believe. If we go this way we have to make sure that collector won't fail to start if that directory doesn't exist -- Please test this scenario. Ideally it should be a warning message saying that directory is not found.

Reply:
I agree. 
We can keep the journald enabled and use `/var/log/journald` as default. If directory doesnt exist, collector doesnt fail to run, but it doesn't provide any warning as such. 
Will we need to update the receiver/ input to generate any such warning right now? or can be added later?

Reply:
It can be added later. Noop if directory doesn't exist is fine for now.